### PR TITLE
SK Post PED

### PR DESCRIPTION
**Pull request by PohSayKeong**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -89,16 +89,20 @@ CS2103T Laboratory 10:00-12:00
 
 ### Adding a default course : `teach`
 
-Adds a default course to the address book.
+The default course is usually the course that you are currently teaching.
```
Comment (lshaoqin) [inline docs/UserGuide.md:94]: The language here is not consistent with the other commands

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 2]
```diff
@@ -183,9 +177,9 @@ public EditPersonDescriptor(EditPersonDescriptor toCopy) {
         /**
          * Returns true if at least one field is edited.
```
Comment (lshaoqin) [inline src/main/java/seedu/address/logic/commands/EditCommand.java]: Need to change this

**Review by lshaoqin — CHANGES_REQUESTED** [review]
Nice job, I suggested two small changes.

File: docs/UserGuide.md (hunk 1 above)
Comment (PohSayKeong) [reply docs/UserGuide.md:94]: resolved in [b3a68ec](https://github.com/AY2324S1-CS2103T-T10-1/tp/pull/123/commits/b3a68ec98c865c793111441f1611b3629c26c262)

File: src/main/java/seedu/address/logic/commands/EditCommand.java (hunk 2 above)
Comment (PohSayKeong) [reply src/main/java/seedu/address/logic/commands/EditCommand.java]: Resolved in [b3a68ec](https://github.com/AY2324S1-CS2103T-T10-1/tp/pull/123/commits/b3a68ec98c865c793111441f1611b3629c26c262)

**Review by lshaoqin — APPROVED** [review]
LGTM

**Pull request merged**